ocaml/idl/datamodel_errors.ml

```diff
@@ -1198,7 +1198,41 @@ let _ =
     ~doc:"No other cluster host was reachable when joining" ();
 
   error Api_errors.xen_incompatible []
-    ~doc:"The current version of Xen or its control libraries is incompatible with the Toolstack." ()
+    ~doc:"The current version of Xen or its control libraries is incompatible with the Toolstack." ();
+
+  (* repository and updates errors *)
+  error Api_errors.set_repository_in_progress []
+    ~doc:"The operation could not be performed because setting repository is in progress." ();
```

Comment:
`because setting up of the repository` or `because a repository setup is in progress`